integration-cli/docker_api_network_test.go

```diff
@@ -289,15 +282,15 @@ func createNetwork(t *testing.T, config network.CreateRequest, expectedStatusCod
 }
 
 func connectNetwork(t *testing.T, nid, cid string) {
-	resp, _, err := request.Post(testutil.GetContext(t), "/networks/"+nid+"/connect", request.JSONBody(network.ConnectOptions{
+	resp, _, err := request.Post(testutil.GetContext(t), "/networks/"+nid+"/connect", request.JSONBody(client.NetworkConnectOptions{
 		Container: cid,
 	}))
 	assert.NilError(t, err)
 	assert.Equal(t, resp.StatusCode, http.StatusOK)
 }
 
 func disconnectNetwork(t *testing.T, nid, cid string) {
-	config := network.ConnectOptions{
+	config := client.NetworkDisconnectOptions{
```

Comment:
Found this probably should have been disconnect options, so fixed it while I had the hood open. LMK if you would like to see this split into a separate commit and backported.

Reply:
heh. nah, should be fine; I'm guessing it worked because it only used that single field, which marshalled fine. But we can consider fixing it in the 28.x branch if we're concerned.